src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/AvroHelper.java

```diff
@@ -39,6 +39,8 @@
 /** A helper to convert sql result set to avro format and dump the avro result to output stream. */
 public class AvroHelper {
 
+  private static final Pattern TRAILING_SPACES_REGEX = Pattern.compile("\\s++$");
```

Comment:
Sorry if I'm getting this wrong, but isn't that regex equivalent to `"\\s+$"`?

Reply:
Have no idea how I caught this file in my CL.
Will rebase origin/dev onto origin/main

Reply:
I think it's coming from rebasing.